qt/python/mantidqt/widgets/tableworkspacedisplay/presenter.py

```diff
@@ -157,11 +156,7 @@ def action_delete_row(self):
         selected_rows_list = [index.row() for index in selected_rows]
         selected_rows_str = ",".join([str(row) for row in selected_rows_list])
 
-        DeleteTableRows(self.model.ws, selected_rows_str)
-        # Reverse the list so that we delete in order from bottom -> top
-        # this prevents the row index from shifting up when deleting rows above
-        for row in reversed(selected_rows_list):
```

Comment:
It is no longer needed to remove the row from the view, as the DeleteTableRows alg triggers a replaceWorkspace ADS event